ldms/src/contrib/sampler/geopm_sampler/Plugin_geopm_sampler.man

```diff
@@ -0,0 +1,173 @@
+load name=ldms_geopm_sampler
+config name=ldms_geopm_sampler producer=${HOSTNAME} geopm_request_path=/etc/ldms/geopm_sampler_request.config
+start name=ldms_geopm_sampler interval=1000000
+.fi
+
+
+Here's an example of a file containing the list of signals:
+
+    $> cat geopm_sampler_signal.config
+    CPU_FREQUENCY_MAX board 0
+    CPU_FREQUENCY_MIN board 0
+    CPU_FREQUENCY_STEP board 0
+    CPU_FREQUENCY_STICKER board 0
+    TIME board 0
+    ENERGY_PACKAGE board 0
+    INSTRUCTIONS_RETIRED board 0
+    POWER_DRAM board 0
+    POWER_PACKAGE board 0
+    POWER_PACKAGE_LIMIT board 0
+    POWER_PACKAGE_MAX board 0
+    POWER_PACKAGE_MIN board 0
+    POWER_PACKAGE_TDP board 0
+    TEMPERATURE_CORE board 0
+    TEMPERATURE_PACKAGE board 0
+    TIMESTAMP_COUNTER board 0
+
+
+Note the inclusion of the
+.IR geopm_request_path " parameter passed to the " config 
+instruction. Also, note the name of the sampler
+.IR ldms_geopm_sampler
+passed to the
+.IR name 
+parameter for the
+.IR load 
+and 
+.IR start 
+instructions. 
+
+
+.B RUNNING LDMSD WITH THE SAMPLER
+
+In order to run the GEOPM LDMS sampler, follow the same steps as you would 
+for any other LDMS sampler.  Start the ``ldmsd`` daemon is running on 
+the target node to be monitored. Example below:
+
+    ldmsd -x sock:10444 -F -c <path-to-sampler-conf-script> -l ${TEST_PATH}/temp/demo_ldmsd_log 
+
+
+For observing the progress of the sampler, you may choose to add the 
+option ``-v DEBUG`` above. While the ``ldmsd`` daemon is running, the user 
+may choose to query for a single instantaneous sample set comprising of 
+recently monitored signals.  This can be achieved by using the existing 
+commandline tool - ``ldms_ls`` available as part of the installation of 
+the LDMS framework. An example is shown below:
+
+     $> ldms_ls -h localhost -x sock -p 10444 -l -v
+
+     Schema         Instance                 Flags  Msize  Dsize  Hsize  UID    GID    Perm       Update            Duration          Info
+     -------------- ------------------------ ------ ------ ------ ------ ------ ------ ---------- ----------------- ----------------- --------
+     ldms_geopm_sampler <hostname>/ldms_geopm_sampler    CL    1352    240      0   1024    100 -r--r----- 1656431193.051578          0.000323 "updt_hint_us"="1000000:50000"
+     -------------- ------------------------ ------ ------ ------ ------ ------ ------ ---------- ----------------- ----------------- --------
+     Total Sets: 1, Meta Data (kB): 1.35, Data (kB) 0.24, Memory (kB): 1.59
+     
+     =======================================================================
+     
+     <hostname>/ldms_geopm_sampler: consistent, last update: Tue Jun 28 08:46:33 2022 -0700 [51578us]
+     M u64        component_id                               1
+     D u64        job_id                                     0
+     D u64        app_id                                     0
+     D d64        CPU_FREQUENCY_MAX-board-0                  3700000000.000000
```

Comment:
Hi Guys,

Although '-' is present in a lot of other schema, could we replace them with '_'? The reason is that if we store these items in the database and then try to query it with a NoSQL syntax, the '-' is a special character that we might want to use in the future for server side operations.

We're trying to get away from these special characters in metric names. Specifically, '(', ')', '-', '+', etc...